src/appShell/App/PortalHeader.tsx

```diff
@@ -102,31 +105,25 @@ export default class PortalHeader extends React.Component<{}, {}> {
                 <Link to="/" id="cbioportal-logo"><img src={require("./cbioportal_logo.png")} alt="cBioPortal Logo"/></Link>
                 <nav id="main-nav">
                     <ul>
-
                         {
                            this.getTabs()
                         }
-
-
                     </ul>
                 </nav>
             </div>
 
             <div id="rightHeaderContent">
-                <If condition={!_.isEmpty(AppConfig.authUserName)}>
+                <If condition={this.props.appStore.isLoggedIn}>
                     <Then>
-                        <div className="identity">Logged in as <span dangerouslySetInnerHTML={{__html:AppConfig.authUserName!}}></span>
+                        <div className="identity">Logged in as {this.props.appStore.userName}
```

Comment:
what on earth was going on with the way it used to be and dangerouslySetInnerHTML

Reply:
not sure.  i'm hoping that there would never be any HTML in there.  but who the heck knows!